### PR TITLE
Allow to use the deploy template with non-WordPress projects

### PR DESCRIPTION
**Pull request by ocean90**

### CODE REVIEW

File: res/deploy.tpl.php [hunk 1]
```diff
@@ -67,13 +76,15 @@ function () {
 task(
 	'wp:opcache_clear',
 	function () {
+		if ( ! get( 'wp_clear_opcache' ) ) {
```
Comment (derpaschi) [inline res/deploy.tpl.php]: As we're running wp commands here, shouldn't we check for the new `wordpress` flag as well? For when `wp_clear_opcache` got forgotten to delete.
```suggestion
		if ( ! get( 'wp_clear_opcache' ) || ! get( 'wordpress' ) ) {
```

File: res/deploy.tpl.php [hunk 2]
```diff
@@ -49,6 +54,10 @@ function () {
 task(
 	'wp:update_translations',
 	function () {
+		if ( ! has( 'wp_languages' ) ) {
```
Comment (derpaschi) [inline res/deploy.tpl.php]:
```suggestion
		if ( ! has( 'wp_languages' ) || ! get( 'wordpress' ) ) {
```

File: res/deploy.tpl.php [hunk 3]
```diff
@@ -33,6 +34,10 @@ function () {
 task(
 	'wp:install_translations',
 	function () {
+		if ( ! has( 'wp_languages' ) ) {
```
Comment (derpaschi) [inline res/deploy.tpl.php]:
```suggestion
		if ( ! has( 'wp_languages' ) || ! get( 'wordpress' ) ) {
```

File: composer.json [hunk 4]
```diff
@@ -28,7 +28,8 @@
   },
   "require-dev": {
     "composer/composer": "1.6.* || 2.0.*@dev",
-    "composer/semver": "^1 || ^3"
+    "composer/semver": "^1 || ^3",
+    "deployer/deployer": "^6.8"
```
Comment (derpaschi) [inline composer.json:32]: No need to required deployer here, or do we?

**Review by derpaschi — CHANGES_REQUESTED** [review]

File: res/deploy.tpl.php (hunk 1 above)
Comment (ocean90) [reply res/deploy.tpl.php]: Not sure about this, why would someone set `wp_clear_opcache` after `wordpress` is set to false?

File: composer.json (hunk 4 above)
Comment (ocean90) [reply composer.json:32]: It's added as a dev dependency for autocomplete support etc..

File: res/deploy.tpl.php (hunk 1 above)
Comment (derpaschi) [reply res/deploy.tpl.php]: Just for when somebody forgets to set this to false. Same with `wp_languages` for example.

Comment (ocean90) [reply res/deploy.tpl.php]: You don't have to set it to false. The option should be omitted if not used.

**Review by derpaschi — DISMISSED** [review]

**Review by derpaschi — APPROVED** [review]
LGTM 👌🏻

**Pull request merged**